CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Contributing to Force
```

Comment:
https://github.com/artst/force => https://github.com/artsy/force in a few places here